sound/soc/sof/ipc4-pcm.c

```diff
@@ -368,10 +375,11 @@ static int sof_ipc4_trigger_pipelines(struct snd_soc_component *component,
 					       trigger_list);
 	}
 
+skip_pause_transition:
 	/* return if this is the final state */
 	if (state == SOF_IPC4_PIPE_PAUSED)
 		goto free;
-skip_pause_transition:
+
```

Comment:
it's not clear from the commit message if this duplicate IPC causes the firmware to go into a weird state, or it this patch is just removing a useless command.